sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py

```diff
@@ -886,27 +887,27 @@ def list_properties_of_certificate_versions(
         account_id = kwargs.pop("account_id", None)
         password = kwargs.pop("password", None)
         organization_id = kwargs.pop("organization_id", None)
-        admin_details = kwargs.pop("admin_details", None)
+        admin_contacts = kwargs.pop("admin_contacts", None)
 
         if account_id or password:
             issuer_credentials = self._client.models.IssuerCredentials(account_id=account_id, password=password)
         else:
             issuer_credentials = None
-        if admin_details and admin_details[0]:
-            admin_details_to_pass = list(
+        if admin_contacts and admin_contacts[0]:
+            admin_contacts_to_pass = list(
                 self._client.models.AdministratorDetails(
                     first_name=admin_detail.first_name,
                     last_name=admin_detail.last_name,
                     email_address=admin_detail.email,
                     phone=admin_detail.phone,
                 )
-                for admin_detail in admin_details
+                for admin_detail in admin_contacts
             )
         else:
-            admin_details_to_pass = admin_details
-        if organization_id or admin_details:
+            admin_contacts_to_pass = admin_contacts
```

Comment:
This confuses me. The overall intent seems to be, if `admin_contacts` is a list, translate it to a list of `AdministratorDetails`. But if we don't get something iterable, the `else` obtains and we pass along whatever unexpected argument we got?

Reply:
yeah this check doesn't make sense as a whole, I'm going to just check to see if it exists and if not, just pass None